xonsh/ptk/completer.py

```diff
@@ -2,6 +2,7 @@
 """Completer implementation to use with prompt_toolkit."""
 import os
 import builtins
+import xonsh
```

Comment:
No worries, not wasting anyone's time.  We are always happy to have more xonsh developers :)